.gitignore

```diff
@@ -0,0 +1,9 @@
+.idea
```

Comment:
Use this boiler plate .gitignore then add to it if needed.
https://github.com/github/gitignore/blob/master/Python.gitignore